CONTRIBUTING.md

```diff
@@ -84,3 +84,34 @@ npm --prefix aiida_workgraph/web/frontend cache clean
 ```
 
 and also clear your browsers cache or try to start new private window.
+
+
+### Building the docs
+
+We use sphinx to build the docs. You need the requirements in the extra
+`.[docs]` dependency and the `docs/requirements.txt`. We have a `docs/Makefile`
+that runs sphinx-build to build the docs.
+
+```console
+pip install .[docs]
+pip install -r docs/requirements.txt
+make -C docs html
+<YOUR-BROWSER> docs/build/html/index.html
+```
+
+#### Creating an with executable code
+For executable code we create first jupyter notebook with that should be
+integrated into the docs. Then we transform it the notebook to a sphinx-gallery
+script (a python file that can be parsed by sphinx-gallery to generate an
+`.rst`.) using this
```

Comment:
```suggestion
For executable code with use sphinx-gallery to integrate into the doc. For that we need create a sphinx-gallery script (a python file that can be parsed by sphinx-gallery to generate an
`.rst`) instead of a `.rst` file. One can create a sphinx-gallery script from a jupyter notebook using the
```